platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/LongListHeap.java

```diff
@@ -180,4 +186,10 @@ protected void partialChunkCleanup(
     protected AtomicLongArray createChunk() {
         return new AtomicLongArray(numLongsPerChunk);
     }
+
+    @Override
+    public void close() {
```

Comment:
Not only this is redundant (see my proposal above about closing the temp buffer after the long list is completely read from the file), but it's simply wrong. Current temp buffer is a static field, it must not be deleted, when any instance of `LongListHeap` is closed. This is one more reason to make the field non-static.

Need a test case to cover this.